src/config.cpp

```diff
@@ -107,7 +134,7 @@ void parse_program_options(int argc, const char** argv)
              "suppress output")
         ("verbose,v", po::value(&verbosity)->zero_tokens(),
              "verbose output (specify multiple times to get increasingly more verbose output)")
-        ("mmap-pages,m", po::value(&config.mmap_pages)->default_value(16),
+        ("mmap-pages,m", po::value(&config.mmap_pages)->default_value(get_default_mmap_value()),
```

Comment:
In accordance with the previous change, use `get_perf_event_mlock() - get_page_size()` as default.